Source/WebKit/Shared/API/APIData.h

```diff
@@ -71,7 +70,7 @@ class Data : public ObjectImpl<API::Object::Type::Data> {
     {
         auto bufferSize = buffer.size();
         auto bufferPointer = buffer.releaseBuffer().leakPtr();
-        return createWithoutCopying({ bufferPointer, bufferSize }, [] (uint8_t* bytes, const void*) {
+        return createWithoutCopying(unsafeForgeSpan(bufferPointer, bufferSize), [] (uint8_t* bytes, const void*) {
```

Comment:
Should we give Vector a releaseSpan() function?

Reply:
Yes, I will look into making this factoring in my next PR.